examples/rstudio/test_rstudio.py

```diff
@@ -0,0 +1,33 @@
+import getpass
+import time
+import selenium
+from selenium import webdriver
+from selenium.webdriver import FirefoxOptions
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
```

Comment:
Our style is to `import` rather than `from ... import`. You can `import ... as ...` if you need a shorthand.